modules/removeListitem.js

```diff
@@ -0,0 +1,12 @@
+import updateBookData from './updateBookData.js';
+
+export default function removeListitem(index, toDoList) {
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍